resources/definitions/fdmprinter.def.json

```diff
@@ -5130,7 +5130,7 @@
                 "support_z_distance":
                 {
                     "label": "Support Z Distance",
-                    "description": "Distance from the top/bottom of the support structure to the print. This gap provides clearance to remove the supports after the model is printed. This value is rounded up to a multiple of the layer height.",
+                    "description": "Distance from the top/bottom of the support structure to the print. This gap provides clearance to remove the supports after the model is printed. Apart from the top-layer, this value is rounded up to a multiple of the layer height.",
```

Comment:
This does not look clear to me... If you have e.g. a layer height 0.2 and Z distance 0.44, you will get 2 "normal" layers and a pseudo 0.04 layer right ? If this is correct, then last part is more confusing than helping.

Reply:
I can see how it might be unclear, do you have a suggestion?

Reply:
Well, I would not talk about the "normal" layers because what they will do seem obvious. So I would just say something like "The topmost layer might be shorter that regular layers to respect the total distance"